examples/megacmd/megacmdexecuter.cpp

```diff
@@ -2749,8 +2763,12 @@ void MegaCmdExecuter::executecommand(vector<string> words, map<string, int> *clf
                     if (words.size() > 2)
                     {
                         path = words[2];
-                        if (fsAccessCMD->isFolder(&path))
+                        string localpath;
```

Comment:
this should better go into a private function (eliminating the complication in the flux when deleting)